spec/support/shared_contexts/api/docs/api_create_endpoint_documentation_support.rb

```diff
@@ -0,0 +1,63 @@
+RSpec.shared_context "an API create endpoint documentation", :exceptions_app do |options = {}|
+  path options[:url] do
+    post "Creates a #{options[:resource_description]}" do
+      tags options[:tag]
+      consumes "application/json"
+      produces "application/json"
+      security [api_key: []]
+
+      parameter name: :params,
+                in: :body,
+                style: :deepObject,
+                required: false,
+                schema: {
+                  "$ref": options[:request_schema_ref],
+                }
+
+      response "200", "The created #{options[:resource_description]}" do
+        schema({ "$ref": options[:response_schema_ref] })
+
+        # after do |example|
```

Comment:
not sure why this is commented out?

Reply:
ah I had brought in [this helper](https://github.com/DFE-Digital/npq-registration/blob/main/spec/support/helpers/swagger_example_parser.rb) that lets us modify examples as I thought it was needed but its not yet, so I pulled it back out and forgot to remove this 🙂 I've removed it now